arch/arm/include/irq.h

```diff
@@ -29,6 +29,12 @@
  * Included Files
  ****************************************************************************/
 
+#include <nuttx/config.h>
+#ifndef __ASSEMBLY__
+#  include <stdbool.h>
+#  include <sys/types.h>
```

Comment:
I think `sys/types.h` can be moved out of `ifndef __ASSEMBLY__`

Reply:
Done